src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/util/Paginate.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.util
+
+import org.springframework.data.domain.PageImpl
+import org.springframework.data.domain.Pageable
+
+infix fun <T> List<T>.paginateWith(pageable: Pageable): PageImpl<T> {
```

Comment:
```suggestion
infix fun <T> List<T>.paginateWith(paginationOptions: Pageable): PageImpl<T> {
```
Same as above, this is clearer to me than `pageable".